src/Python-2.0.1.xml

```diff
@@ -0,0 +1,284 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license licenseId="Python-2.0.1" name="Python Software
+   Foundation License 2.0.1" listVersionAdded="3.18">
+      <crossRefs>
+         <crossRef>https://www.python.org/download/releases/2.0.1/license/</crossRef>
+         <crossRef>https://docs.python.org/3/license.html</crossRef>
+         <crossRef>https://github.com/python/cpython/blob/main/LICENSE</crossRef>
+      </crossRefs>
+      <notes>
+         This is the stack of licenses used for Python version 1.6.1 onward.
+      </notes>
+      <text>
+         <titleText>
+            <p>
+               PSF LICENSE AGREEMENT FOR PYTHON 3.10.6
+            </p>
+         </titleText>
+         <list>
+            <item>
+               <bullet>1.</bullet>
+               This LICENSE AGREEMENT is between the Python Software Foundation
+               ("PSF"), and the Individual or Organization ("Licensee")
+               accessing and otherwise using <alt match="this software ("Python")|Python 3.10.6|Python 2.0.1" name="software"> this software ("Python")</alt>
+               in source or binary form and its associated documentation.
+            </item>
+            <item>
+               <bullet>2.</bullet>
+               Subject to the terms and conditions of this License Agreement,
+               PSF hereby grants Licensee a nonexclusive, royalty-free,
+               world-wide license to reproduce, analyze, test, perform
+               and/or display publicly, prepare derivative works, distribute,
+               and otherwise use Python <alt match="2.0.1|3.10.6| " name="version">2.0.1</alt> alone or in any derivative
```

Comment:
Here and below, I think this would be:

`match="(([0-9]+)\.([0-9]+)\.([0-9]+))?"`

I think this would match either nothing (no version given) or a `1.2.3` style of version numbering.

Side note, for the `name=""` attribute, I don't know whether these need to be different names for each instance here and below... try it and see, I guess  :shrug:

Reply:
tried other solution below